scripts/ci/docker-compose/local.yml

```diff
@@ -109,7 +109,4 @@ services:
       - type: bind
         source: ../../../chart
         target: /opt/airflow/chart
-      - type: bind
```

Comment:
This is the problem - don't remove those lines :)

Reply:
Hmm. I don't know why it got removed. Will add it back

Reply:
Bad merge likely. It's been added and removed in the last few weeks.